.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--
+The RIOT community cares a lot about code quality.
+Therefore, before describing what your contribution is about, we would like
+you to make sure that your modifications are compliant with the RIOT
+coding conventions, see https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions.
+-->
+
+### Contribution description
+
+<!--
+Put here the description of your contribution:
+- describe which part(s) of RIOT is (are) involved
+- if it's a bug fix, describe the bug that it solves and how it is solved
+- you can also give more information to reviewers about how to test your changes
+-->
+
+
+### Issues/PRs references
+
+<!--
+Example: Fixes #1234. See also #5678.
+Please use keywords (e.g., Fixes) to create link to the issues or pull requests
+you resolved, so that they will automatically be closed when your pull request
+is merged. See https://github.com/blog/1506-closing-issues-via-pull-requests.
```

Comment:
PRs usually don't get closed when a referring PR gets merged (i.e. the first sentence just applies to issues). Maybe adapt the wording appropriately here to not be misleading.